packages/flame_test/lib/src/epsilon.dart

```diff
@@ -0,0 +1,110 @@
+import 'dart:typed_data';
+
+/// Returns the next larger representable float32 value from [value].
+///
+/// This is useful for creating test values that are just above a target value.
+/// For example, if you want a value that's just above 1.0, you can use:
+/// `final justAbove1 = getNextFloat32(1.0);`
+double getNextFloat32(double value) {
```

Comment:
Generally in Dart you don't name things `get`/`set`, this could just be `nextFloat32`. :)

Very nice helper methods though!